CHANGES.rst

```diff
@@ -38,6 +38,8 @@ New Features
 
 - ``astropy.io.misc``
 
+  - Yaml representer now accepting also numpy types
```

Comment:
super nitpicking, but could you please add a full stop and the PR number as it's done in the other entries, ``[#6077]``? (We have a few scripts that checks the changelog consistency before release and they use that information)